patterns/dswp-image-text-flipped.php

```diff
@@ -1,33 +1,38 @@
 <?php
```

Comment:
Maybe we should add an HTML formatter package since it's a little hard to see the changes in the patterns when they're formatted this way. I did a Format Document in VSCode and it looks pretty good: 
![image](https://github.com/user-attachments/assets/fae398c5-145b-46a7-9708-422764da0253)

Reply:
@mhaswell-bcgov  I'll make a ticket